dm/proto/dmmaster.proto

```diff
@@ -304,3 +307,23 @@ message CheckTaskResponse {
     bool result = 1;
     string msg = 2;
 }
+
+message FetchSourceInfoRequest {
```

Comment:
How about adding some comments to explain the meaning of fields, and whether they are required or optional?

Reply:
Added.